core/src/main/resources/db/migration/V2_1__alter_release_id.sql

```diff
@@ -0,0 +1,17 @@
+--: ----------------------------------------------------------------------------
+--: Copyright (C) 2017 Verizon.  All Rights Reserved.
+--:
+--:   Licensed under the Apache License, Version 2.0 (the "License");
+--:   you may not use this file except in compliance with the License.
+--:   You may obtain a copy of the License at
+--:
+--:       http://www.apache.org/licenses/LICENSE-2.0
+--:
+--:   Unless required by applicable law or agreed to in writing, software
+--:   distributed under the License is distributed on an "AS IS" BASIS,
+--:   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--:   See the License for the specific language governing permissions and
+--:   limitations under the License.
+--:
+--: ----------------------------------------------------------------------------
+ALTER TABLE PUBLIC."audit_log" ALTER COLUMN "release_id" VARCHAR(25);
```

Comment:
I am not familiar with DB migrations, this will probably need @timperrett 's eyes.